src/ui/components/modals/appModalContainerStyle.ts

```diff
@@ -0,0 +1,47 @@
+import { DefaultTheme } from 'styled-components/native'
+
+import { getSpacing } from 'ui/theme'
+
+const BORDER_HORIZTONAL_RADIUS = getSpacing(5)
+const BORDER_VERTICAL_RADIUS = getSpacing(4)
+
+type Props = {
+  theme: DefaultTheme
+  desktopMaxHeight?: number
+  height?: number
+  maxHeight: number
+}
+
+export const appModalContainerStyle = ({ theme, height, desktopMaxHeight, maxHeight }: Props) => {
+  return {
+    alignItems: 'center',
+    backgroundColor: theme.colors.white,
+    borderTopStartRadius: BORDER_VERTICAL_RADIUS,
+    borderTopEndRadius: BORDER_VERTICAL_RADIUS,
+    borderTopRightRadius: BORDER_HORIZTONAL_RADIUS,
+    borderTopLeftRadius: BORDER_HORIZTONAL_RADIUS,
+    flexDirection: 'column',
+    justifyContent: 'center',
+    padding: getSpacing(6),
+    paddingBottom: getSpacing(8),
+    width: '100%',
+    height: height ?? undefined,
```

Comment:
```ts
let height = undefined // lorsque height vaut undefined
height ?? undefined
```

lorsque `height` vaut `undefined`, tu demandes à ne pas utiliser sa valeur mais préférer utiliser `undefined`

ajouter `??` avec la valeur `undefined`, me semble inutile

on peut donc simplifier

```suggestion
    height: height,
```

qu'on peut encore simplifier

```suggestion
    height,
```